### PR TITLE
Add getCookieParam method

### PR DESCRIPTION
**Pull request by martynbiz**
Consistent with getParam and getQueryParam

### CODE REVIEW

**Comment by dopesong** [comment]
Tests missing :)

**Comment by AlexDanault** [comment]
I'd also add that getCookieParam should be after getCookiesParams and not before, to be consistent with getQueryParam (which is after getQueryParams) and getParsedBodyParam (also after getParsedBodyParams)

**Comment by martynbiz** [comment]
Hi, thanks for the feedback. Sorry, I'm a little new to creating PR. I hope I've done it correctly here - https://github.com/slimphp/Slim/pull/1929 - let me know if there is anything I've missed or done incorrectly. Thanks :)

**Comment by dopesong** [comment]
So you don't need to create new PR, just add new commits to existing branch and PR will be updated automatically. :)

File: Slim/Http/Request.php [hunk 1]
```diff
@@ -670,6 +670,26 @@ public function getContentLength()
...
+    public function getCookieParam($key, $default = null)
+    {
+        $getCookies = $this->getCookieParams();
+        $result = $default;
+        if (isset($getCookies[$key])) {
+            $result = $getCookies[$key];
+        }
+        return $result;
+    }
+
```
Comment (jony4) [inline Slim/Http/Request.php:691]: How about this?

```
    public function getCookieParam($key)
    {
        $cookies = $this->cookies;
        if (isset($cookies[$key])) {
            return $cookies[$key];
        }
        return false;
    }

```

**Comment by akrabat** [comment]
Replaced by #1929

**Pull request closed without merging**